pkg/maps/set.go

```diff
@@ -1,44 +1,46 @@
+// SPDX-FileCopyrightText: 2022 Per Alexander Fougner
```

Comment:
Per? 🤭 How official of you :)

Reply:
There's another Swedish programmer by the name of Alexander Fougner hehe